src/Controller/SentencesListsController.php

```diff
@@ -145,8 +145,14 @@ public function collaborative($filter = null)
      *
      * @return mixed
      */
-    public function show($id = null, $translationsLang = null)
+    public function show($id = null, $lang = null, $translationsLang = null)
     {
+        // this makes me cringe but it works
```

Comment:
I suggest you change this comment into something that explains why the redirection is needed, i.e. backward compatibility with legacy URLs.